src/util/std_expr.cpp

```diff
@@ -211,3 +211,47 @@ const exprt &object_descriptor_exprt::root_object() const
 
   return *p;
 }
+
+bool evaluates_to_boolean(const exprt &e)
```

Comment:
This function is defined here, but not actually used anywhere?

Reply:
Yes, Michael wanted it in `util` but Martin did not want it to be used (in this PR).

Reply:
So maybe it should also be removed from this PR and be added to whichever PR actually does make use of it?